### PR TITLE
IOGT 18:  Only language switch - more that one languages

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@peterp please review 

I have removed the language dropdown is there's only one language available.

Updated:
screenshot 2016-08-22 14 56 44

screenshot 2016-08-22 14 56 56

**Comment by Mitso** [comment]
@peterp please review

File: iogt/templates/base.html [hunk 1]
```diff
@@ -85,26 +85,34 @@
 
       {% if languages %}
       <div class="languages">
+       {% if languages|length <= 1 %}
```
Comment (peterp) [inline iogt/templates/base.html]: you may be able to write this `{% if languages|length_is:"1" %}

See https://docs.djangoproject.com/en/1.10/ref/templates/builtins/#length-is

File: iogt/templates/base.html [hunk 2]
```diff
@@ -166,12 +174,14 @@
               <img src="{% static 'img/footer_login_icon.png' %}" width="30" class="menu-list__item--icon" />{% trans "Profile" %}</a>
             {% endif %}
           </li>
+          {% if languages|length > 1 %}
```
Comment (peterp) [inline iogt/templates/base.html]: Here you can just write it as `{% if languages %}` 

See: http://stackoverflow.com/questions/902034/how-can-i-check-the-size-of-a-collection-with-django-templates

**Comment by Mitso** [comment]
Thanks @peterp.
I updated {% if languages|length_is:"1" %}
The second  option does not work: I'm thinking because if there is 1 language will result  {% if languages %} true

**Comment by peterp** [comment]
👍

**Pull request merged**